build-system/compile/generate/bento.js

```diff
@@ -55,6 +51,35 @@ function generateBentoRuntimeEntrypoint(
   `);
 }
 
+/**
+ * @param {Object<string, string[]>} packageSymbols
+ * @return {string}
+ */
+function generateBentoCoreEntrypoint(packageSymbols) {
+  packageSymbols = packageSymbols || getSharedBentoSymbols();
+
+  assertNoDupes(Object.values(packageSymbols).flat());
+  return dedent(`
+    ${generateEsImports(packageSymbols, true)}
+  `);
+}
+
+/**
+ *
+ * @param {Object<string, string[]>} packageSymbols
+ * @param {boolean} generateExport
+ * @return {string}
+ */
+function generateEsImports(packageSymbols, generateExport = false) {
```

Comment:
This is definitely a nitpick.
It seems strange for a function named `generateEsImports` to also generate exports.
Maybe change the name to something like `generateEsImportExports`?
```suggestion
function generateEsImportExports(packageSymbols, generateExport = false) {
```

Reply:
updated